easybuild/easyconfigs/p/PLINK/PLINK-2.0.0-a.5.15-gfbf-2023b.eb

```
@@ -0,0 +1,58 @@
+easyblock = 'Bundle'
+
+name = 'PLINK'
+version = '2.0.0-a.5.15'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = """PLINK is a free, open-source whole genome association analysis toolset,
+designed to perform a range of basic, large-scale analyses in a computationally
+efficient manner."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+
+dependencies = [
+    ('zlib', '1.2.13'),
+    ('zstd', '1.5.5'),
+    ('libdeflate', '1.19'),
+]
+
+default_easyblock = 'MakeCp'
+
+default_component_specs = {
+    'source_urls': ['https://github.com/chrchang/plink-ng/archive/refs/tags/'],
+    'sources': ['v%s.tar.gz' % version],
+    'skipsteps': ['configure'],
+}
+
+components = [
+    ('PLINK1', '1.90', {
+        'checksums': ['db7bd1ad89bf25c5436f5fbd56caa9f5cc5abd9cf667cb1e7c57eb69396a9992'],
+        'start_dir': '%s-ng-%s/1.9' % (name.lower(), version),
+        'buildopts': (
+            'CFLAGS="${CFLAGS} -DDYNAMIC_ZLIB" CXXFLAGS="${CXXFLAGS} -DDYNAMIC_ZLIB" '
```

```suggestion
            'CFLAGS="${CFLAGS} -O1 -DDYNAMIC_ZLIB" CXXFLAGS="${CXXFLAGS} -DDYNAMIC_ZLIB" '
```

Tests pass for 5.17 with that change.